ruby/tests/gc_test.rb

```diff
@@ -6,7 +6,9 @@
 old_gc = GC.stress
 GC.stress = 0x01 | 0x04
 require 'generated_code_pb'
-require 'generated_code_proto2_pb'
+# TODO(teboring): The following line is commented out for unblocking CI. Add
```

Comment:
I still don't understand why things in this file are commented out. What needs to be fixed? What error messages are we getting now?